dox/dox_only.sh

```diff
@@ -1,5 +1,3 @@
 #!/bin/bash
-
 # Run doxygen-style documentation system.
-
```

Comment:
Here you would set the py_interpreter with the Envvar set by cmake and ic not take the default interpr.